composer.json

```diff
@@ -18,7 +18,7 @@
         "ext-mbstring": "*",
         "behat/gherkin": "^4.5.1",
         "behat/transliterator": "^1.2",
-        "symfony/console": "~2.5||~3.0||~4.0",
+        "symfony/console": "^2.7.40||^3.3.15||^4.0.3",
```

Comment:
If it's desired to restrict the component to versions without this bug we should probably also add`^2.8.33` and `^3.4.3`.

Reply:
Let's revert to `~` then

Reply:
For 2.7 and 3.3 I mean